src/main/java/uk/gov/hmcts/reform/userprofileapi/service/ResourceDeleter.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.userprofileapi.service;
+
+import uk.gov.hmcts.reform.userprofileapi.controller.response.UserProfilesDeletionResponse;
+import uk.gov.hmcts.reform.userprofileapi.resource.RequestData;
+
+public interface ResourceDeleter<T extends RequestData> {
+
```

Comment:
rename this to DeleteResourceService

Reply:
done. followed the same old structure but removed now.